packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -300,4 +313,27 @@
             right: 16px;
         }
     }
+    &--filter {
+        margin-left: 1rem;
+        margin-top: 0;
+        max-width: fit-content;
+        @include mobile {
+            height: 4rem;
+        }
+    }
+}
+
+.icon_filter {
+    position: absolute;
+    left: 10px;
+    @include mobile {
```

Comment:
ditto

Reply:
This one is targetting the icon( not native picker)